src/System.CommandLine.ApiCompatibility.Tests/LocalizationTests.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Globalization;
+using System.Linq;
+using Xunit;
+
+namespace System.CommandLine.ApiCompatibility.Tests
```

Comment:
I've added this type to a different project than `System.CommandLine.Tests` to ensure that it does not use the resources included in that project.